math/wxMaxima/Portfile

```diff
@@ -72,9 +84,9 @@ wxMaxima and Maxima startup files can typically be found in:
 ATTENTION: On macOS BigSur there is an issue with wxWidgets, which has the\
 effect that wxMaxima does not start. In order to fix this, run these\
 commands:
-  sudo port -f uninstall wxwidgets-3.0
-  sudo port -v -s install wxwidgets-3.0
-If this does not help, you need to update XCode.
+  sudo port -f uninstall wxWidgets-3.2
+  sudo port -v -s install wxWidgets-3.2
+If this does not help, you need to update Xcode.
```

Comment:
@MSoegtropIMC What was the problem with the prebuilt wxWidgets 3.0 archive that prompted the recommendation to build it from source? If wxWidgets 3.0 still has a problem such that it builds wrong with some Xcode versions, that needs to be fixed; if so, is there at least a MacPorts ticket for that problem?

Was the problem fixed in wxWidgets 3.2? If so, then the note should be removed—or maybe not: wxMaxima still depends on wxWidgets 3.0 via its dependency gnuplot.

Reply:
@ryandesign : it was an XCode toolchain problem - the binary version was built with a broken version of XCode. This was fixed by installing a new version of XCode on the MacPorts build servers and rebuilding all binaries a while back.

Reply:
Users can install any version of Xcode and can build ports from source. If a specific range of Xcode versions produces a broken build of wxWidgets-3.0 or wxWidgets-3.2, the ports need to issue an error rather than build something broken. The usual way bad versions of Xcode would be prevented is using the xcodeversion portgroup. Better yet, if the problem is related to the compiler version and not to Xcode, the usual way bad compiler versions would be prevented is using the compiler_blacklist_versions portgroup. I see neither of those portgroups used in the wxWidgets-3.0 or wxWidgets-3.2 ports. So if there is not a ticket about this, please file one so it can be fixed properly.

Reply:
The ticket on wxWidgets I created back then is this one:

https://trac.macports.org/ticket/62348

And related:

https://trac.macports.org/ticket/61634

As far as I can see you where involved in fixing it, so you should know better than me :-)

Reply:
Ohhhh that. The versions of Xcode with the [broken `install_name_tool`](https://lists.macports.org/pipermail/macports-dev/2020-November/042641.html). MacPorts should probably just [refuse to install anything](https://trac.macports.org/ticket/67788) if one of those versions is installed.

Reply:
Indeed refusing this version of Xcode in general is the best option.